setup.cfg

```diff
@@ -43,14 +43,14 @@ console_scripts =
 [options.extras_require]
 doc =
     furo
-    sphinx
-    sphinx-autodoc-typehints
-    sphinxcontrib-autoprogram
+    sphinx~=3.0
+    sphinx-autodoc-typehints~=1.10
+    sphinxcontrib-autoprogram~=0.1.0
 test =
-    filelock
-    pytest
-    pytest-cov
-    pytest-mock
+    filelock~=3.0
```

Comment:
Ok then, let's make this to ``>= 3``. We can keep the docs part to ``~=``.

Reply:
Okay.

Reply:
This change hasn't been addressed yet.

Reply:
Sorry, forgot to push :upside_down_face: